frontend/indigenousapprenticeship/src/js/components/Home/Home.tsx

```diff
@@ -78,20 +84,17 @@ const Home: React.FunctionComponent = () => {
           data-h2-offset="p-tablet(5%, auto, auto, 50%)"
           data-h2-width="base(100%) p-tablet(40vw)"
         >
-          <img
-            data-h2-width="base(100%)"
-            src={imageUrl(
-              INDIGENOUSAPPRENTICESHIP_APP_DIR,
-              `logo-${intl.locale}.svg`,
-            )}
-            alt={intl.formatMessage({
-              defaultMessage:
-                "IT Apprenticeship Program for Indigenous Peoples",
-              id: "Hu04cP",
-              description:
-                "Homepage title for Indigenous Apprenticeship Program",
-            })}
-          />
+          <h1>
+            <img
+              data-h2-width="base(100%)"
+              src={imageUrl(
+                INDIGENOUSAPPRENTICESHIP_APP_DIR,
+                `logo-${intl.locale}.svg`,
+              )}
+              alt={pageTitle}
+            />
+            <span data-h2-visibility="base(invisible)">{pageTitle}</span>
```

Comment:
Having both an `alt` and a hidden `span` will lead to repetition of the text. This falls under the [image with adjacent text alternative](https://www.w3.org/WAI/tutorials/images/decorative/#example-3-image-with-adjacent-text-alternative) scenario for decorative images.

Reply:
Oh okay thanks Eric. I'll make a fix for that and remove the alt.

Reply:
Fixed by [0125d47](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4867/commits/0125d47d59f2480091c9e9925f69d618308ded4d)